config/database.yml

```diff
@@ -12,7 +12,8 @@ default: &default
 
 development:
   <<: *default
-  database: marc_liberation_dev
+  # database: marc_liberation_dev
+  database: marc_liberation_alma_dev
```

Comment:
@carolyncole is this change necessary

Reply:
@christinach no, thanks for catching that...  I removed it.